airbyte-integrations/connectors/destination-s3-data-lake/src/test-integration/kotlin/io/airbyte/integrations/destination/s3_data_lake/S3DataLakeWriteTest.kt

```diff
@@ -249,3 +251,52 @@ class NessieMinioWriteTest :
         }
     }
 }
+
+@Execution(ExecutionMode.SAME_THREAD)
+class RestWriteTest : S3DataLakeWriteTest(getConfig(), NoopDestinationCleaner) {
+    @Test
+    @Disabled("https://github.com/airbytehq/airbyte-internal-issues/issues/11439")
+    override fun testFunkyCharacters() {
+        super.testFunkyCharacters()
+    }
+
+    override val manyStreamCount = 5
+
+    @Disabled(
+        "This just does not seem to work with the rest catalog provided by apache as a quick start. Seems to not like any sort of concurrency"
+    )
+    @Test
+    override fun testManyStreamsCompletion() {
+        super.testManyStreamsCompletion()
+    }
+
+    companion object {
+        fun getConfig(): String {
+            val minioEndpoint = RestTestContainers.testcontainers.getServiceHost("minio", 9000)
+            val restEndpoint = RestTestContainers.testcontainers.getServiceHost("rest", 8181)
+
+            return """
+            {
+                "catalog_type": {
+                  "catalog_type": "REST",
+                  "server_uri": "http://$restEndpoint:8181",
+                  "namespace": "test"
```

Comment:
fyi - after https://github.com/airbytehq/airbyte/pull/53186 merges, we should change this to `"namespace": "<DEFAULT_NAMESPACE_PLACEHOLDER>"`